app.js

```diff
@@ -109,7 +109,6 @@ routes.integrations(route('/integrations/'));
 routes.github(route('/github/'));
 
 routes.react(route('/react/'));
-routes.ember(route('/ember/'));
```

Comment:
Can you change this to a redirect so we don't end up with link rot? I think there's a util for this.